src/globus_cli/commands/timer/create/transfer.py

```diff
@@ -108,6 +98,15 @@ def resolve_optional_local_time(
     help="Stop running the transfer after this number of runs have happened.",
 )
 @mutex_option_group("--stop-after-date", "--stop-after-runs")
+@click.option(
+    "--delete",
```

Comment:
I wonder if this should be named "--delete-destination-extra" instead of simply "--delete" to keep the explict-ness.

Additionally, in the help section, it's unclear what "extraneous files in the destination directory" means. Perhaps, using what's in the [transfer docs](https://docs.globus.org/api/transfer/task_submit/#recursive_directory) would be better?

> Used to provide "mirroring" by deleting any files in the destination directory not in the source.

Reply:
Oh just read your comment again and saw you chose specifically to use `--delete`.

Reply:
Yeah, I definitely think that `delete-destination-extra` is a much better flag than `delete`. I just don't want this command to diverge at all from the other one.

How about we add this & I'll file a story to deprecate `--delete` in favor of `--delete-destination-extra`. We'll need to support a long deprecation tail on that keyword no matter what but this lets us sync up the commands as they are right now.

---

For the docs, I'm happy to update. I'll update the docs for `globus transfer --delete` as well though

Reply:
Updated the shorttext in [3524f82](https://github.com/globus/globus-cli/pull/1017/commits/3524f820843c9c9a027f33bd1347c3849b98971b). 

It's not identical to the transfer docs you linked cause it's missing some of the surrounding context in that style of doc. Lemme know what you think.